src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/unusedimports/InputUnusedImportsFromStaticMethodRefJavadocDisabled.java

```diff
@@ -0,0 +1,93 @@
+/*
+UnusedImports
+processJavadoc = false
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.imports.unusedimports;
+
+import static java.lang.annotation.ElementType.TYPE_USE; //ok
+import static java.lang.annotation.RetentionPolicy.RUNTIME; //ok
+
+import java.lang.annotation.Retention; //ok
+import java.lang.annotation.Target; //ok
+import java.lang.ref.WeakReference; //ok
+import java.util.ArrayList; //ok
+import java.util.Optional; //ok
+import java.util.Objects; //ok
+import java.util.Arrays; //ok
+import java.util.List; //ok
+import java.util.Map; //ok
+import java.util.Collections; //ok
+import java.util.function.Consumer; //ok
+import java.util.function.Function; //ok
+import java.util.function.Supplier; //ok
```

Comment:
please remove all `// ok` , no comment is ok by default.
we sometime keep `// ok ..... reason as it  might be not obvious ....`